mpt-witness-generator/witness/prepare_witness.go

```diff
@@ -116,7 +115,10 @@ func obtainAccountProofAndConvertToWitness(i int, tMod TrieModification, tModsLe
 	} else if tMod.Type == BalanceChanged {
```

Comment:
Line 99: adding "be"  `	// This needs to be called before oracle.PrefetchAccount, otherwise oracle.PrefetchAccount
`